### DISCUSSION TITLE
Random Number Component

### DISCUSSION THREAD

**Discussion by PrismMind** [FakeFishGames/Barotrauma#6320] [Ideas and suggestions]
As the title says, It would make life alot easier if we could have a Random number Component that when receive signal it outputs a random number between x and y.

**Comment by Misterzelix** [comment]
+1 would allow for a lot more easily made or in-depth systems.

**Comment by civilCornball** [comment]
Don't we already have one of these?
Try using an Oscillator component on Sine mode, with the frequency on incredibly high.
As long as you don't constantly use its output, that's already a random number generator that looks between [-1, 1].

**Reply by PrismMind** [reply]
Sure but I do want a constant output as well

**Comment by PsykokiN** [comment]
+1 need this

**Comment by Gesugao-san** [comment]
Any updates?…
